Sources/SwiftSyntax/Raw/RawSyntax.swift

```diff
@@ -774,3 +774,11 @@ extension RawSyntax {
   }
 }
 
+extension RawUnexpectedNodesSyntax {
```

Comment:
I just added a similar extension as well but I think this should live in the parser module, because really it’s a parser-specific use case. I created a SyntaxUtils.swift file to hold it.

Reply:
Let me know when you've got that one in. No real rush on this PR. I haven't even done the diagnostics yet :)(

Reply:
It's in now
https://github.com/apple/swift-syntax/blob/d30f30d95e9854b25ae8f51b6380ebec6fe41cb4/Sources/SwiftParser/SyntaxUtils.swift#L27-L35